### ISSUE TITLE
Feature request: add support for Pushover

### ISSUE THREAD

**Issue by tpjanssen** [dennissiemensma/dsmr-reader#246]
Dear Dennis,

Is it possible to add support for Pushover? My guess is that it would be fairly simple to do so.

Regards.

**Comment by dennissiemensma** [comment]
Hi, I asume you refer to the notification service Pushover? (Googled it)

**Comment by tpjanssen** [comment]
Sorry, for the incomplete request, but yes, that's what I would like to see implemented.

Thanks.

**Comment by dennissiemensma** [comment]
Dit is nodig als vervanger van Notify My Android. De service werkt helaas wel iets anders, namelijk net zoals Dropbox. Gebruikers moeten zelf een applicatie registreren en dan zowel een API-key als user-token invoeren.

**Comment by dennissiemensma** [comment]
Ja, klopt. En die komt vrij snel, ik probeer ze wat kleiner te houden tegenwoordig.

**Comment by dennissiemensma** [comment]
Bij deze is v1.18 uitgebracht.
